frontend/server/src/DAO/Problems.php

```diff
@@ -553,10 +553,16 @@ public static function getProblemsSolvedCount(\OmegaUp\DAO\VO\Identities $identi
             INNER JOIN
                 Problems p ON p.acl_id = a.acl_id
             WHERE
-                p.visibility = ? AND
+                (
+                    p.visibility >= ? OR
+                    p.visibility = ?
+                ) AND
                 i.identity_id = ?;';
 
-        $params = [\OmegaUp\ProblemParams::VISIBILITY_PUBLIC, $identityId];
+        $params = [
+            \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING,
+            \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_BANNED,
+            $identityId];
```

Comment:
nit:

```suggestion
            $identityId,
        ];
```

intentamos que todos los arreglos se cierren en una línea aparte si no cupieron en una sóla línea. y siempre que se pueda se agrega la coma al final de cada elemento por consistencia.

Reply:
@lhchavez en que test existente puedo agregar la prueba?